Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -1120,6 +1143,11 @@ def get_test_list_and_content_packs_to_install(files_string, branch_name, two_be
     if 'NonSupported' in packs_to_install:
         packs_to_install.remove("NonSupported")
 
+    # get all modified packs - not just tests related
+    modified_packs = get_modified_packs(files_string)
```

Comment:
Lets avoid adding `NonSupported` pack in case it was modified. So you can move 
```
  if 'NonSupported' in packs_to_install:
        packs_to_install.remove("NonSupported")
```
to the end. Or ignore appending this pack in `get_modified_packs` function.

Reply:
Good catch. moved to the end